solidity/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@keep-network/keep-ecdsa",
-  "version": "1.7.0-pre",
+  "version": "1.7.0-rc",
```

Comment:
This doesn't match the version in `package.json`.

Reply:
Fixed, see a19133e4c3804a745aa30f65d9ab204df0640d57.